boards/arm/nucleo_wb55rg/doc/nucleo_wb55rg.rst

```diff
@@ -189,6 +189,9 @@ Projects/STM32WB_Copro_Wireless_Binaries/STM32WB5x/
 For compatibility information with the various versions of these binaries,
 please check `modules/hal/stm32/lib/stm32wb/hci/README <https://github.com/zephyrproject-rtos/hal_stm32/blob/main/lib/stm32wb/hci/README>`__
 in the hal_stm32 repo.
+Note that since STM32WB Cube package V13.2.0, "full stack" binaries are not compatible
```

Comment:
do you refer to stm32wb  cube version V**1.13.2** ?

Reply:
Nice catch, thanks.